src/main/java/org/mskcc/cbio/oncokb/service/UserService.java

```diff
@@ -247,6 +248,9 @@ public User registerUser(UserDTO userDTO, String password) {
         userRepository.findOneByEmailIgnoreCase(userDTO.getEmail()).ifPresent(existingUser -> {
             throw new EmailAlreadyUsedException();
         });
+        userRepository.findOneByLogin(userDTO.getLogin().toLowerCase()).ifPresent(existingUser -> {
+            throw new LoginAlreadyUsedException();
+        });
```

Comment:
A user's login is the same as their email, so the first check should be sufficient. However, the login check is added in case that is changed.